totp/__init__.py

```diff
@@ -20,6 +22,33 @@ def get_length(pass_entry):
     return 6
 
 
+def add_pass_entry(path):
+    """Add a new entry via pass."""
+    code_path = "2fa/{}/code"
+    code_path = code_path.format(path)
+
+    token_length = input('Token length [6]: ')
+    token_length = int(token_length) if token_length else 6
+
+    shared_key = getpass.getpass('Shared key: ')
+
+    pass_entry = "digits: {}\n{}".format(token_length, shared_key)
```

Comment:
I meant "`shared_key` needs to be the first line [in the file you're writing]".

In other words, the lines you're writing need to be swapped. You're currently writing:

```
digits: 6
SHARED_KEY
```

When it should actually be:
```
SHARED_KEY
digits: 6
```